src/services/routes/Local.js

```diff
@@ -53,8 +52,16 @@ export async function readFile(filepath) {
  * @param {String} filepath Full file path to delete
  * @param {String} content Content to write to given file path
  */
-export async function writeFile(filepath, content) {
-  return RNFS.writeFile(filepath, content, 'utf8')
+export async function writeFile(isImage, filepath, content) {
```

Comment:
just as a heads up: i've changed this in my code, both in terms of conciseness and adding an extra term to deal with image encoding